README.rst

```diff
@@ -332,7 +332,7 @@ What is the software license?
    :target: https://heroku.com/deploy?template=https://github.com/fossasia/susi_server/tree/development
 .. |Deploy on Scalingo| image:: https://cdn.scalingo.com/deploy/button.svg
    :target: https://my.scalingo.com/deploy?source=https://github.com/fossasia/susi_server
-.. |Deploy to Docker Cloud| image:: https://files.cloud.docker.com/images/deploy-to-dockercloud.svg
+.. |Deploy to Docker Cloud| image:: https://user-images.githubusercontent.com/59795301/72219546-3f6c7000-356d-11ea-977c-c6be5770d423.png
```

Comment:
This does not actually allow to deploy on docker. The other buttons allow for deployment on various clouds directly, while this button also shows the image without any useful target. It would be good to fix the deploy target, too.